static-site/src/components/People/avatars.jsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import styled from "styled-components";
+import { teamMembers } from "./teamMembers";
+
+const CommonWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  & span {
+    white-space: nowrap;
+  }
+  & img {
+    margin-left: 5px;
+    border-radius: 50%;
+    vertical-align: middle;
+  }
+  & a {
+    font-weight: 500 !important;
+    color: #333 !important;
+  }
+`
+
+const FooterWrapper = styled(CommonWrapper)`
+  & img {
+    margin-right: 4px;
+    width: 32px;
+    height: 32px;
+  }
+`
+
+const BodyWrapper = styled(CommonWrapper)`
+  padding: 20px 0px;
+  & span {
+    font-size: 18px;
+  }
+  & img {
+    margin-right: 20px;
+    width: 60px;
+    height: 60px;
+  }
+  & > div {
+    padding: 10px 15px;
+  }
+  & img > div {
+    font-size: 50px;
+  }
+`
+
+const Sideways = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-start;
+  align-items: center;
+`
+
+const AvatarContainer = styled(Sideways)`
+  max-width: 40%;
+`
+
+const UpDown = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: column;
+  justify-content: center;
+`
+
+const Blurb = styled.div`
+  font-size: 14px;
+  text-align: justify;
```

Comment:
```suggestion
  text-align: left;
```

![Screen Shot 2022-12-21 at 11 24 02](https://user-images.githubusercontent.com/79913/208986858-1ac8be1c-7fe2-44a1-80e9-6adc5f2de2e4.png)
vs.
![Screen Shot 2022-12-21 at 11 23 40](https://user-images.githubusercontent.com/79913/208986865-3d5f1228-61f8-47ed-be65-fe4b92d71786.png)

Reply:
Made as 7200e4c given "Feel free to modify styling as you see fit :)" :-)